### DISCUSSION TITLE
Limit Recent Saved Searches

### DISCUSSION THREAD

**Discussion by dd-manderson** [burkybang/SuiteAdvanced-Feedback#120] [Ideas]
Hello!  Excited to be using SuiteAdvanced and looking forward to participating in the community.

When viewing the "Recent Saved Searches", it would be nice if it limited those searches to ones that just I edited or executed.  You would not include ones I created but were executed by others.  Right now I'm seeing saved searches that were executed by everyone in the system.

**Comment by burkybang** [comment]
Welcome!

The criteria I'm using to show the recent saved searches is `Last Run By = Current User` OR `Last Modified By = Current User`.

I agree it seems to be showing too many, but I'm not sure how to improve it.

If you want to help, you could try creating a saved search of saved searches and play around with the criteria. If you find something that works much better, I'd love to know!

**Reply by burkybang** [reply]
@dd-manderson I did some more testing, and I think I've improved it a bunch. It should be much better in the next release.
